studio/views.py

```diff
@@ -108,6 +113,84 @@ def profile(request):
     return render(request, "user/profile.html", {"user_profile": user_profile})
 
 
+@login_required
+def delete_account(request):
+    """
+    Renders a form that allows a user to delete their user account.
+    Verifies that the user does not own any Serve projects.
+    """
+    logger.debug("Rendering page delete a user account.")
+    logger.info(f"User views page to delete their user account. User {request.user}")
+
+    # Check if the user owns any projects
+    n_projects = Project.objects.filter(Q(owner=request.user), status="active").count()
+
+    if n_projects == 0:
+        account_can_be_deleted = True
+        logger.debug("User account CAN be deleted. The user does not own any projects.")
+    else:
+        account_can_be_deleted = False
+        logger.info(f"User account cannot be deleted. The user owns {n_projects} projects.")
```

Comment:
Maybe use the fact that 0 is cast a False, like this? (only if you want to)

```suggestion
    if n_projects:
        account_can_be_deleted = False
        logger.info(f"User account cannot be deleted. The user owns {n_projects} projects.")
    else:
        account_can_be_deleted = True
        logger.debug("User account CAN be deleted. The user does not own any projects.")
```

Reply:
In my opinion the more explicit == 0 is more readable